### PR TITLE
[DE] Fix responses for light

### PR DESCRIPTION
**Pull request by steffenrapp**
Fix responses for light and cover so that there are no more unused responses and that they all have response checks. Therefore some sentences for covers had to be added as well.

### CODE REVIEW

File: responses/de/HassLightSet.yaml [hunk 1]
```diff
@@ -2,7 +2,5 @@ language: de
 responses:
   intents:
     HassLightSet:
-      brightness: "{{ slots.name }} Helligkeit auf {{ slots.brightness }} gesetzt"
-      brightness_area: Helligkeit in {{ slots.area }} auf {{ slots.brightness }} gesetzt
-      color: "{{ slots.name }} Farbe auf {{ slots.color }} gesetzt"
-      color_area: Farbe in {{ slots.area }} auf {{ slots.color }} gesetzt
+      brightness: "Helligkeit eingestellt"
+      color: "Farbe auf {{ slots.color }} gesetzt"
```
Comment (mib1185) [inline responses/de/HassLightSet.yaml:6]: meanwhile I think (_based on the [Response Style Guide](https://developers.home-assistant.io/docs/voice/intent-recognition/style-guide)_) we should not repeat any input data (_area, name, state, color,..._)

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @steffenrapp 
thanks for bring this in 👍 
Nevertheless, please split this PR by `light` and `cover` and remove the repeating of names/areas/states in the corresponding responses.

File: responses/de/HassLightSet.yaml (hunk 1 above)
Comment (steffenrapp) [reply responses/de/HassLightSet.yaml]: Done.

**Comment by steffenrapp** [comment]
Hi @mib1185 😃 I removed the cover changes from this PR and will create a separate PR once this one is merged. Please go on with your review 👍

File: responses/de/HassTurnOff.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ responses:
   intents:
     HassTurnOff:
       default: "{{ slots.name }} ausgeschaltet"
-      lights_area: "Die Lampen in {{ slots.area }} sind ausgeschaltet"
+      lights_area: "Licht ausgeschaltet"
```
Comment (mib1185) [inline responses/de/HassTurnOff.yaml:6]: we should call it just `light`

**Review by mib1185 — CHANGES_REQUESTED** [review]
"technically"  fine 👍  but just one naming should be adjusted

File: responses/de/HassTurnOff.yaml (hunk 2 above)
Comment (steffenrapp) [reply responses/de/HassTurnOff.yaml]: okay, I changed that

**Comment by steffenrapp** [comment]
@mib1185 please continue, hope everything is fine now 😃

**Review by mib1185 — APPROVED** [review]
LGTM , thx @steffenrapp 👍

**Pull request merged**